src/components/organisms/topbar/project/buttons.tsx

```diff
@@ -208,15 +245,36 @@ export const ProjectTopbarButtons = () => {
 
 			<DropdownButton
 				contentMenu={
-					<Button className="group h-8 px-4 text-white" onClick={openModalDeleteProject} variant="outline">
-						<IconSvg
-							className="-mt-0.5 stroke-white transition group-hover:stroke-green-200 group-active:stroke-green-800"
-							size="md"
-							src={TrashIcon}
-						/>
-
-						{t("topbar.buttons.delete")}
-					</Button>
+					<>
+						<Button
+							ariaLabel={t("topbar.buttons.export")}
+							className="group h-8 px-4 text-white"
+							onClick={exportProject}
+							variant="outline"
+						>
+							<IconSvg
+								className="fill-white transition group-hover:stroke-green-200 group-active:stroke-green-800"
```

Comment:
hover and active color not working, I think remove `stroke` from svg icon

Reply:
changed to `fill`